### PR TITLE
minor english grammar

### PR DESCRIPTION
**Pull request by ghinks**
Minoring grammar change in the readme for mode, 

change from 
according the mode
to
according to the mode

### CODE REVIEW

File: src/content/concepts/mode.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ module.exports = {
 }
 ```
 
-If you want to change the behavior according the **mode** variable inside the *webpack.config.js* you have to export a function instead of an object:
+If you want to change the behavior according to the **mode** variable inside the *webpack.config.js* you have to export a function instead of an object:
```
Comment (Legends) [inline src/content/concepts/mode.md]: The change is fine! I would also change`variable` to `property`, because it's not a variable.

Comment (ghinks) [reply src/content/concepts/mode.md]: yes agreed

**Comment by montogeek** [comment]
@ghinks Please sign CLA.

**Comment by ghinks** [comment]
I have signed the CLA, not sure about the CI build for a markdown change. Do I have to do anything more here or will one of the approving reviewers pick this up?

**Comment by montogeek** [comment]
CLA is still not signed, your commit author email doesn't match your GitHub user email.

**Comment by ghinks** [comment]
ok thanks montogeek, I am going to close this and re-open the commit.

**Comment by ghinks** [comment]
closing and re-opening again with a new PR as it looks like my CLA is not correct.

**Pull request closed without merging**